pkgs/development/libraries/libiscsi/default.nix

```diff
@@ -13,8 +13,13 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook ];
 
+  # This can be removed after >=1.20.0, or if the build suceeds with
+  # pie enabled (default on Musl).
+  hardeningDisable = [ "pie" ];
```

Comment:
Shouldn't this only be disabled on musl?

Reply:
I think the implication is that `pie` of any sort breaks it, but it's just that it's on by default on musl, which reveals the problem.

Reply:
Yep, for instance it needs to be disabled unconditionally in https://github.com/NixOS/nixpkgs/pull/104091